cocotb/monitors/__init__.py

```diff
@@ -113,6 +113,13 @@ def add_callback(self, callback):
                        callback.__name__)
         self._callbacks.append(callback)
 
+    def log_transaction(self, transaction):
+        """Helper to easily log a received transaction
+
+        Typical usage: :any:`moni_obj.add_callback(moni_obj.log_transaction)`
```

Comment:
```suggestion
        Typical usage: ``moni_obj.add_callback(moni_obj.log_transaction)``
```
(since you don't want a hyperlink)